app/views/tasks/show.html.erb

```diff
@@ -0,0 +1,5 @@
+<p> <%= @task.description %> and should be completed 
```

Comment:
this is very much small potatoes but the phrase 'and should be complete' doesn't make a ton of sense here.

also, it would be nice to see the title of the task. :)